lib/improver/tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -457,30 +402,6 @@ def test_multiple_realizations(self):
             [0.91666667, 0.875, 0.91666667])
         self.assertArrayAlmostEqual(result.data, expected)
 
-    def test_multiple_realizations_and_times(self):
```

Comment:
Does this need to be adapted to just be multiple times so we test that, or is there another multiple times (without multiple realizations) test elsewhere?

Reply:
Okay.